packages/material-ui/src/MenuList/MenuList.test.js

```diff
@@ -45,4 +45,76 @@ describe('<MenuList />', () => {
       assert.strictEqual(wrapper.find('div').length, 2);
     });
   });
+
+  describe('actions: adjustStyleForScrollbar', () => {
+    it('should not adjust style when container element height is greater', () => {
+      const menuListActionsRef = React.createRef();
+      const wrapper = mount(<MenuList actions={menuListActionsRef} />);
+      const list = wrapper.getDOMNode();
```

Comment:
Should be possible to attach either `ref` or `innerRef` to `MenuList` and use `.current` as the DOM node. While it doesn't affect the test it is necessary for tests in StrictMode since enzymes `getDOMNode` is not StrictMode ready.